docker/kb-importer/conf/kaybeeconf-initial.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: "v1"
+
+# List of sources from which statements are pulled
+# Ranks: smaller value means higher rank (higher priority)
+sources:
+  - repo: https://github.com/sumeetpatil/project-kb
```

Comment:
https://github.com/sap/project-kb

Reply:
Yes, I will change this 👍 It was just used for testing. Thank You